scripts/install-kusion.sh

```diff
@@ -22,7 +22,7 @@ for arg in "$@"; do
     fi
```

Comment:
Using cat with a pipe is unnecessary here. The base64 command can directly read the file, making the code more efficient and simpler. Consider using 'base64 -w 0 <path>' instead.